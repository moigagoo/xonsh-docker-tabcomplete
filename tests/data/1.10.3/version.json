{
        "arguments": [
                "-f", "--format",
                "--help"
        ],
        "commands": []
}
