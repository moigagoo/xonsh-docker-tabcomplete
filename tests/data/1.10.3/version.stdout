
Usage:	docker version [OPTIONS]

Show the Docker version information

  -f, --format       Format the output using the given go template
  --help             Print usage
